istioctl/pkg/validate/validate.go

```diff
@@ -171,16 +260,27 @@ func NewValidateCommand() *cobra.Command {
 	var referential bool
 
 	c := &cobra.Command{
-		Use:     "validate -f FILENAME [options]",
-		Short:   "Validate Istio policy and rules",
-		Example: `istioctl validate -f bookinfo-gateway.yaml`,
-		Args:    cobra.NoArgs,
+		Use:   "validate -f FILENAME [options]",
+		Short: "Validate Istio policy and rules",
+		Example: `
+		# Validate bookinfo-gateway.yaml
+		istioctl validate -f bookinfo-gateway.yaml
```

Comment:
are we producing any output upon a successful validation?  Just ran it with 1.1.x's, nothing is written out to users.

Reply:
@linsun Unfortunately a commit on April 29 removed the output for successful validation.  I intend to restore the output on success, see https://github.com/istio/istio/issues/14238

Reply:
I have fixed this in this PR to report validation successful.

Reply:
I also made that change in https://github.com/istio/istio/pull/14434